src/main/java/com/adobe/testing/s3mock/util/AwsChunkDecodingInputStream.java

```diff
@@ -38,8 +38,6 @@
  * [hex-encoded-number-of-bytes-in-chunk];chunk-signature=[sha256-signature][crlf]
  * [payload-bytes-of-this-chunk][crlf]
  * </pre>
- *
- * @see com.amazonaws.auth.AwsChunkedEncodingInputStream
```

Comment:
Could you exchange this one with a link to http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/AwsChunkedEncodingInputStream.html?

Reply:
Sure, I've added the link.